pythran/config.py

```diff
@@ -72,13 +70,21 @@ def parse_define(define):
     here = os.path.dirname(os.path.dirname(__file__)) or '.'
     # using / as separator as advised in the distutils doc
     extension["include_dirs"].append(here + '/pythran')
+
+    extra.pop('language', None)  # forced to c++ anyway
+    cxx = extra.pop('cxx', None)
+    if cxx is not None:
+        extension['cxx'] = cxx
+    elif cfg.get('compiler', 'CXX'):
+        extension['cxx'] = str(cfg.get('compiler', 'CXX'))
+
```

Comment:
It maybe useful to add some sort of warning to the user (`DeprecationWarning` ?), to use the custom build_ext class when `CXX` is configured.